test/pixelcopy_test.py

```diff
@@ -185,6 +198,7 @@ def iter_surface_to_array_3d(self, rgba_masks):
                 )
             view = None
 
+    @_suppress_warnings(DeprecationWarning)
     def test_map_array(self):
```

Comment:
Could the function be removed from this unrelated test instead of the deprecation warning being suppressed here?

Reply:
I don't really see any way how it would be possible. Just simply removing ``color.set_length(3)`` breaks the test completely and I don't know how to get change the code that use it, so it will work. Also I don't think removing that part of the check would be a good idea. But, if you have any suggestions, I'm open.

Reply:
color[:3] ?

Reply:
@Starbuck5 "ValueError: tuple object does not export an array buffer", I tried it already...

Reply:
``np.array(color[:3])`` resolves that issue.